Gemfile

```diff
@@ -38,8 +38,8 @@ gem 'momentjs-rails', '>= 2.9.0' # needed for human-friendly textual dates
 gem 'font-awesome-rails'
 
 # needed to run graders in the background
-gem 'delayed_job', github: 'dsander/delayed_job', branch: 'rails51' # FIXME: github
```

Comment:
Steno (the other repo) provides real-time in browser output for grading scripts. This needs backburner on the rails side for instant async execution.

This pull request could be conceptually split into (use the new grading script protocol) and (move to backburner), which are basically orthogonal, but both pieces are needed for Steno.